strings/fishing_tips.txt

```diff
@@ -0,0 +1,36 @@
+Fish scanning experiments can be automatically performed with an advanced fishing rod. However to print one you need to have at least completed the first one.
+Advanced fishing rods come pre-equipped with a unexhaustible bait that ignores bait preferences of the fish, letting you catch'em all.
+Don't expect to be able to fish much without a bait.
+You can fish at the holodeck beach, if you don't mind the fish being quite fake.
+You may find worms by digging through sand and snow.
+You can revive fish by using a Lazarus Injector on them. However, using Strange Reagent would be a smarter option here.
+You can feed fish outside of an aquarium by tapping them with a can of fish feed.
+More fishing rods and fish cases can be printed at the autolathe.
+Seeking alternative ways to catch fish without bothering to do it yourself? Explosives can be thrown at fishing spots to get several (dead) ones.
+You can raise lobstrosities from chasm chrabs. However, only juvenile lobstrosities can be tamed with spare limbs or lavaloop fish.
+Lavaloop fish make for dangerous yet somewhat throwing weapons against big fauna.
+The fishing portal generator has different modules, all of which can be unlocked by completing fish scanning experiments... except for the Syndicate one...
+Several fish have different traits, some of which can influence on how to catch them. Carnivore fish will ignore dough balls and herbivore ones anything that's not from botany.
+Telescopic fishing rods can be bought from cargo.
+Once grown from chrabs and tamed, lobstrosities can be heeded to fish on fishing spots for you.
+Aquariums can be upgraded to bioelectricity generators can a specific kit. From there, you can add electric-generating like the anxious zip zap to generate power.
+Getting better at fishing will net you some small additional advantages, such as receiving more information when examining a fish or a fishing spot.
+Size and weight of a fish can influence of many reagents and fillets you can get out of them, their force as a wapon and how slow they can be to carry around.
+While most fish make for shoddy weapons, a few, like the swordfish and the chainsawfish, can be quite powerful if grown right.
+Fish can be sold to cargo. While most don't sell for over a couple hundred credits, the biggest ones can quite well sell for over 1k each.
+The meat of fish like swordfish, salmon and pufferfish (weren't they poisonous) make for quality ingredients that boost the quality of the end product.
+Stingrays are known for their stinger, which they use to deliver venom to hostiles.
+Some species of fish can be bred into new species under the right conditions.
+Most fish don't survive outside water, so get them somewhere safe like an aquarium or a fish case, or even a moisture trap?!
+No matter how you look at it, most people won't care about fishing. Think of it as a little funny niche.
```

Comment:
```suggestion
No matter how you look at it, most people won't care about fishing. Don't let that stop you. They're just jealous.
```

We should cement these in the game world. Talking about 'niches' kind of feels a little too OOC.